anime-list-master.xml

```diff
@@ -24033,7 +24033,7 @@
   <anime anidbid="8692" tvdbid="259640" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Sword Art Online</name>
     <mapping-list>
-      <mapping anidbseason="0" tvdbseason="0">;10-0;11-0;12-0;</mapping>
+      <mapping anidbseason="0" tvdbseason="0" start="1" end="9" offset="0"/>
```

Comment:
I believe this won't unmap specials 10-12. Not positive on it, I'm open to be proven wrong

Reply:
Same thing here, if '\<mapping anidbseason="0" tvdbseason="0">' will already try to map all eps one-to-one, then change should be reverted but 13-0 added.

Reply:
Although I will point out I only did because I saw other examples of offset="0", so I thought that was sometimes necessary to use; see anidbid="6080" for example.

Reply:
`offset=0` is just a safety net, I've done it several times myself because I don't trust it to always be assumed as the default

Reply:
As per conversation above, 10-13 shouldn't need unmapping if HAMA works as expected, so the removal of unmapping should be fine (unless you want to leave it in as safety net).

Reply:
Fine by me, if it breaks the `git blame` is on you anyways 🤣